### DISCUSSION TITLE
Requires support for the variable checker to display dataframe row names

### DISCUSSION THREAD

**Discussion by EraShinji** [posit-dev/positron#4315] [Feedback] [answered]
While analyzing some of the table data for conversion to a Seurat object, I found that positron's variable inspector does not currently support displaying the row names of the data frames, forcing me to go back to rstudio to continue my work. Hopefully support for displaying row names can be added to positron's variable inspector.
Best wishes.

**Comment by juliasilge** [comment, marked as answer]
Can you share a screenshot of which part of the product you would like to see row names displayed in? We have https://github.com/posit-dev/positron/issues/4120 where we are tracking row names in what we call the Data Explorer, but you may be referring to the Variables pane?

**Reply by EraShinji** [reply]
Thanks for your reply, and, right, I want to be able to view the row names in the data explorer.
